dev/plotExprTrajectory.R

```diff
@@ -0,0 +1,50 @@
+# plotExprTrajectory.R
+
+#' Title.
+#'
+#' \code{plotExprTrajectory} Plots an iGraph of the p-Creod with coloring of the gene expression
+#'
+#' Details.
+#' @section Dependency: the pcreode package, which can be installed with:
+#' \code{sudo pip install pcreode}
```

Comment:
Sorry - won't merge code with external dependencies. Too bad.

Reply:
If the purpose of the package is to interact with an external package, I am having trouble seeing why it would be a problem.

Reply:
*pardon me, the purpose of the function

Reply:
I'll review this independently ... but we need to adhere to the principle that all dependencies need to be satisfied from CRAN or Bioconductor. It's a nice tool as far as I can see, but not suitable for this package. I'll have a more detailed look tomorrow.

Reply:
... could you close the request and delete from Travis queue? Just ensure there is a link to your fork on your Student Wiki page.

Reply:
Will do